FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -9,6 +9,7 @@
 import UIKit
 import Combine
 
+@MainActor
```

Comment:
`@MainActor` on type makes all methods and properties to be isolated so we could avoid marking each method with `@MainActor`.